src/thumbnailer.h

```diff
@@ -84,6 +86,11 @@ class Thumbnailer {
 
   Status SetLoopCount(WebPData* const webp_data);
 
+  // Takes two thumbnails as WebPData. Prints various statistics regarding
+  // the differences in PSNR between two thumbnails.
+  Status CompareThumbnail(WebPData* const webp_data_ref,
```

Comment:
`CompareThumbnail()`, `AnimData2Pictures()` and `AnimData2PSNR()` should not be part of `Thumbnailer` if they are in a separate library. I advise separate functions in a compare.h, utils.h or similar because they are not really related to `Thumbnailer` (but may be in the same library for convenience).